### DISCUSSION TITLE
Ct100 skippping CC queries during interview

### DISCUSSION THREAD

**Discussion by in2slack** [zwave-js/zwave-js#1760] [General] [answered]
During the interview process for my CT100 thermostat some of the CC queries are being skipped which is preventing me from being able to see and control fan state.  All other functionality seems OK.  Is this a bug or something that has yet to be implemented?  

14:57:40.424 CNTRLR [Node 046] skipping query for Clock (0x81) because max implemented version i
s 1
14:57:40.774 CNTRLR [Node 046] skipping query for Thermostat Fan Mode (0x44) because max impleme
nted version is 0
14:57:40.775 CNTRLR [Node 046] skipping query for Thermostat Fan State (0x45) because max implem
ented version is 0

The device seem to be detected correctly....

14:57:39.722 CNTRLR » [Node 046] querying manufacturer information...
2021-02-14 14:57:39.841 INFO ZWAVE: Node 46: value added: 114-0-manufacturerId => 152
2021-02-14 14:57:39.844 INFO ZWAVE: Node 46: value added: 114-0-productType => 25601
2021-02-14 14:57:39.846 INFO ZWAVE: Node 46: value added: 114-0-productId => 262
14:57:39.854 CNTRLR « [Node 046] received response for manufacturer information:
manufacturer: Radio Thermostat Company of America (RTC) (0x98)
product type: 0x6401
product id: 0x0106

**Comment by AlCalzone** [comment, marked as answer]
Are you on zwave-js 6.3.0? The Thermostat Fan CCs were literally just merged and released today.

**Reply by in2slack** [reply]
I am using what is available in supervised HomeAssistant... i think that is 6.2.0.  Sounds like I just need to wait for update to be made available.  

I appreciate the quick response and hard work.
